example/run-vms.go

```diff
@@ -69,7 +69,7 @@ func getFullConfig() qcli.Config {
 			qcli.BlockDevice{
 				Driver:    qcli.PFlash,
 				ID:        "pflash1",
-				File:      "uefi_nvram.fd",
+				File:      "uefi-nvram.fd",
```

Comment:
Let's leave these as they are; it's an unrelated change to enabling tpm on arm support